src/theme/index.js

```diff
@@ -1,11 +1,35 @@
-// Starting Theme
+import darkTheme from './darkTheme'
+import lightTheme from './lightTheme'
+
+const localTheme = localStorage.getItem('localTheme')
```

Comment:
I don't know if I missed it, but I don't see where you have set a theme into local storage